### PR TITLE
Cleanup configs and make them understandable

### PR DESCRIPTION
**Pull request by imbstack**
As it says on the tin

### CODE REVIEW

File: services/hooks/config.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ defaults:
   # defaults outlining all configuration options and some sane defaults
   app:
     component:        hooks
-    hookTableName:     !env HOOK_TABLE_NAME
-    lastFireTableName: !env LASTFIRE_TABLE_NAME
-    queuesTableName:  !env QUEUE_TABLE_NAME
+    hookTableName: Hooks
+    lastFireTableName: LastFire3
```
Comment (djmitche) [inline services/hooks/config.yml]: Will this require a corresponding change to the static client?  And, do I recall that changes to the static clients don't get deployed automatically?

File: dev-docs/dev-config-example.yml [hunk 2]
```diff
@@ -150,9 +150,6 @@ github:
   pulse_username: ...
```
Comment (djmitche) [inline dev-docs/dev-config-example.yml]: note for later, but this file should probably be ignored in .gitattributes and .git/CODEOWNERS?

File: services/hooks/config.yml (hunk 1 above)
Comment (imbstack) [reply services/hooks/config.yml]: Why will changing the table name require updating clients? I think I'm missing something or other here.

Comment (djmitche) [reply services/hooks/config.yml]: The hooks client has scope `auth:azure-table:read-write:${azureAccountId}/LastFire`, not `auth:azure-table:read-write:${azureAccountId}/LastFire3`

Comment (imbstack) [reply services/hooks/config.yml]: Oh, those clients! haha, I was thinking of every other client for some reason. Yes, good catch.

**Pull request merged**